src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  // height: 406px;
+  width: 200px;
+  background: $color-background;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  transition: transform 300ms;
+  padding-inline: 16px;
+  padding-bottom: 16px;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__product {
+    padding: 0;
+    margin-bottom: 40px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 0;
+  }
+
+  &__label {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #060b35;
+    width: 100%;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-family: Roboto, sans-serif;
```

Comment:
Don't repeat `font-family`. Set it to the html tag and inherit where it is needed